IdentityCore/src/MSIDError.h

```diff
@@ -171,6 +171,8 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
      */
 
     MSIDErrorServerUnhandledResponse    = -51500,
+    // http status Code 403 or 404
+    MSIDErrorUnExpectedHttpResponse     = -51501,
```

Comment:
nit: typo `unexpected`
```suggestion
    MSIDErrorUnexpectedHttpResponse     = -51501,
```

Reply:
Fixed